system/modules/core/library/Contao/Picture.php

```diff
@@ -258,16 +259,27 @@ protected function getTemplateDataSource($imageSize)
 				// Use pixel density descriptors if the sizes attribute is empty
 				if (empty($imageSize->sizes))
 				{
-					$src .= ' ' . $density . 'x';
+					if ($fileObj->width && $file1x->width)
```

Comment:
In the first iteration the descriptor will be `1x`, but in all further iterations it won’t be `1x` as `$fileObj->width` and `$file1x->width` are not the same then.